### PR TITLE
Fix scripts

### PR DESCRIPTION
**Pull request by wendigo**

### CODE REVIEW

File: action.yml [hunk 1]
```diff
@@ -107,42 +107,42 @@ runs:
...
-          MAVEN_RELEASE_PUSH_COMMITS="${{ inputs.mavenReleasePushCommits }}"
-          MAVEN_RELEASE_EXECUTE_POST_SCRIPTS="${{ inputs.mavenReleaseExecutePostScripts }}"
-          MAVEN_RELEASE_EXECUTE_PRE_SCRIPTS="${{ inputs.mavenReleaseExecutePreScripts }}"
-          MAVEN_RELEASE_ARGUMENTS="${{ inputs.mavenReleaseArguments }}"
-          GIT_USER_NAME="${{ inputs.gitUserName }}"
-          GIT_USER_EMAIL="${{ inputs.gitUserEmail }}"
-          MAVEN_RELEASE_TAG="${{ inputs.mavenReleaseTag }}"
-          MAVEN_RELEASE_VERSION="${{ inputs.mavenReleaseVersion }}"
-          MAVEN_NEXT_DEVELOPMENT_VERSION="${{ inputs.mavenNextDevelopmentVersion }}"
+        RELEASE_POST_SCRIPTS=${{ inputs.mavenPostScripts }}
```
Comment (losipiuk) [inline action.yml:110]: why removing quotes? Does `${{` take care of it? Or is it yaml syntax? (sorry for being noob :) )

File: action.yml [hunk 2]
```diff
@@ -107,42 +107,42 @@ runs:
...
+        MAVEN_RELEASE_TAG=${{ inputs.mavenReleaseTag }}
+        MAVEN_RELEASE_VERSION=${{ inputs.mavenReleaseVersion }}
+        MAVEN_NEXT_DEVELOPMENT_VERSION=${{ inputs.mavenNextDevelopmentVersion }}
         EOF
     - name: "Set GIT automation credentials"
       run: git config --global user.name "${{ inputs.gitUserName }}" && git config --global user.email "${{ inputs.gitUserEmail }}"
       shell: bash
+    - name: "Export release version"
+      id: version
+      run: ${{ github.action_path }}/version.sh
```
Comment (losipiuk) [inline action.yml]: rename to export_version.sh?

File: action.yml [hunk 3]
```diff
@@ -107,42 +107,42 @@ runs:
...
+      shell: bash
     - name: "Check release prerequisites"
       run: ${{ github.action_path }}/check.sh
       shell: bash
     - name: "Run pre actions"
       run: ${{ github.action_path }}/pre.sh ${{ inputs.mavenReleaseArguments }}
       shell: bash
-    - name: "Export release version"
```
Comment (losipiuk) [inline action.yml:142]: Oh this is just a move :)

File: action.yml (hunk 1 above)
Comment (wendigo) [reply action.yml:110]: heredoc preserve quotes (`echo key="${value}"` doesn't) and they were printed to file which was incorrect. no quotes needed there

File: action.yml (hunk 2 above)
Comment (wendigo) [reply action.yml]: oki

**Review by losipiuk — APPROVED** [review]

File: action.yml (hunk 3 above)
Comment (wendigo) [reply action.yml:142]: Yeah, that's required for pre actions to be able to read version

File: action.yml (hunk 1 above)
Comment (losipiuk) [reply action.yml:110]: so the variable (like `inputs.mavenPreScripts`) has the quotes already in the value?

**Pull request merged**